pyproject.toml

```diff
@@ -35,14 +35,15 @@ dev = [
   "pytest-cov",
   "pre-commit"
 ]
-chemformer = ["syntheseus-chemformer"]
-local-retro = ["syntheseus-local-retro"]
-megan = ["syntheseus-megan"]
-mhn-react = ["syntheseus-mhnreact"]
-retro-knn = ["syntheseus-local-retro", "torch-scatter"]
-root-aligned = ["syntheseus-root-aligned"]
+chemformer = ["syntheseus-chemformer==0.1.1"]
+graph2edits = ["syntheseus-graph2edits==0.1.0"]
+local-retro = ["syntheseus-local-retro==0.2.0"]
+megan = ["syntheseus-megan==0.1.0"]
+mhn-react = ["syntheseus-mhnreact==1.0.0"]
+retro-knn = ["syntheseus[local-retro]", "torch-scatter"]
+root-aligned = ["syntheseus-root-aligned==0.1.0"]
```

Comment:
Exact pins are needed because the single-step model repos are controlled by us and thus they are effectively a part of `syntheseus` (so there is no point in having the model versions be _not_ tied to `syntheseus` versions). If we did not pin, it would make future changes to single-step model repos disruptive to previously released `syntheseus` versions, e.g. if we release `syntheseus-megan 0.2.0` which includes breaking changes, all previous versions of `syntheseus` would automatically start pulling in the newer version of `syntheseus-megan`, whereas our intention would be for that new version to only affect the respectively updated version of `syntheseus`.

Because `syntheseus 0.3.0` did not pin, the just released version of `syntheseus-local-retro` does break LocalRetro in `syntheseus 0.3.0`. We could potentially add backwards compatibility hacks to prevent this, but I think it's better to just release `syntheseus 0.4.0` soon

Reply:
Actually, I think there is no breakage of `syntheseus 0.3.0`. I thought there was, but I made a mistake when releasing `syntheseus-local-retro`, which I now fixed. Nevertheless, the pins above are still useful, as in the future we may not be so lucky and an update to a single-step model repo may end up being breaking.

Reply:
That's a good idea. Looks good Kris!